<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopCheckClass</class>
 <widget class="QMainWindow" name="PopCheckClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PopCheck</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextIn">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>151</height>
         </size>
        </property>
        <property name="plainText">
         <string>曾桥
江一凡
张浩芫
卢睿
杨家乐
西加艾提
易沐容
王思翔
高雅哲
邵嘉鹏
张珂
王源
庞世龙
刘永浩
朱晨希
魏宇璇
胡茜妮
张诗怡
吉若凡
刘学易
侯一鸣
章玉希
王铭哲
邓尧姿
陈玉玲
张国庆
蔡宇阳
邵乐迪
方俊华</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextCheck">
        <property name="plainText">
         <string>1张浩芫50
2刘学易30
3高雅哲30
4江一凡67
5杨家乐60
6朱晨希40
7邵嘉鹏45
8吉若凡30
9邓尧姿30
10魏宇璇45
11刘永浩45
12王源34
13陈玉玲67
14西加艾提40
15张珂40
16蔡宇阳30
17王思翔 67
18邵乐迪30
19王铭哲35
20方俊华60
21曾桥35
22卢睿40
23庞世龙20+20
24章玉希67
25易沐容63
26张诗怡50
27张国庆37
28侯一鸣50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextOut">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="documentTitle">
         <string>Who</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionCheck"/>
   </widget>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PopCheck.qrc"/>
 </resources>
 <connections/>
</ui>
